c2/c2-client-bundle/README.md

```diff
@@ -0,0 +1,61 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+## Apache NiFi MiNiFi Command and Control (C2) Client
+The c2-client-bundle provides implementation for the client aspect of the [C2 Protocol](https://cwiki.apache.org/confluence/display/MINIFI/C2+Design). The essence of the implementation is the heartbeat construction and the communication with the [C2 server](../../../../minifi-c2/README.md) via the C2HttpClient.
+
+Relying on the [C2 Protocol API](../c2-protocol) functionality right now is limited to sending heartbeats and processing/acknowledging UPDATE configuration operation in the response (if any). Whem exposed the new configuration will be downloaded and passed back to the system using the C2 protocol.
+
+C2ClientService needs to be scheduled to send heartbeats periodically, so the C2 Server can notify the client about any operations that is defined by the protocol and needs to be executed on the client side.
```

Comment:
Suggestion: `When C2 is enabled, C2ClientService will be scheduled to send heartbeats periodically...`

Reply:
Applied